client/src/main/java/org/spine3/change/DoubleMismatch.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2016, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.spine3.change;
+
+import static org.spine3.protobuf.Values.pack;
+
+/**
+ * Utility class for working with {@code int} values in {@link ValueMismatch}es.
+ *
+ * @author Alexander Yevsyukov
+ */
+public class DoubleMismatch {
+
+    private DoubleMismatch() {
+        // Prevent instantiation of this utility class.
+    }
+
+    /**
+     * Creates a ValueMismatch instance for a double attribute.
+     *
+     * @param expected the value expected by a command
+     * @param actual   the value actual in an entity
+     * @param newValue the value from a command, which we wanted to set instead of {@code expected}
+     * @param version  the current version of the entity
+     * @return new ValueMismatch instance
+     */
+    public static ValueMismatch of(double expected, double actual, double newValue, int version) {
```

Comment:
Do we intentionally avoid `Double` and use `double` instead? What if the expected value is `null`?
If we could use a type instead of a primitive, it should then be possible to have a type hierarchy with a generic type, e.g. 
`NumberMismatch<TYPE extends Number>` 
and `DoubleMismatch extends NumberMismatch<Double>`

In this way we could save up some effort by reusing `unpacking methods`.

The same question applies to every `...Mismatch` except for `boolean`.